lib/rules/sort-imports.js

```diff
@@ -112,15 +117,25 @@ module.exports = {
                 return node.specifiers[0].local.name;
             }
             return null;
+        }
 
+        /**
+         * Gets the number of lines between the given declarations.
+         * @param {ASTNode} node1 the ImportDeclaration node.
+         * @param {ASTNode} node2 the ImportDeclaration node.
```

Comment:
This function assumes that `node1` appears after `node2` in the source code, so it would be nice to rename these params to make it clear. Maybe `declaration` and `previousDeclaration` ?